docs/guides/references/troubleshooting.mdx

```diff
@@ -228,6 +228,15 @@ Cloud.
 - `https://download.cypress.io` - **CDN download of Cypress binary**
 - `https://on.cypress.io` - **URL shortener for link redirects**
 
+**If you are using GitHub Enterprise or GitLab for Enterprise (Self-managed),
```

Comment:
Oh, I wasn't aware of this document originally I guess.  Thanks for adding the IPs here 🥳 

Could you also add the capture url above in this same file as well?
```
`https://capture.cypress.io` - **Cypress Test Replay**
```

Reply:
same on this one